Datasets/data-connect-dataset-groupdetails.md

```diff
@@ -0,0 +1,142 @@
+{"id":"2e4b3b4a-6f37-466c-a21b-976ad4518e5a","deletedDateTime":null,"classification":null,"createdDateTime":"2021-10-20T06:25:25Z","description":null,"displayName":"DeniedUsersForPdtDvtGen2","expirationDateTime":null,"groupTypes":[],"isAssignableToRole":null,"mail":"deniedusersforpdtdvtgen2@contosotest4.onmicrosoft.com","mailEnabled":true,"mailNickname":"deniedusersforpdtdvtgen2","membershipRule":null,"membershipRuleProcessingState":null,"onPremisesDomainName":null,"onPremisesLastSyncDateTime":null,"onPremisesSyncEnabled":null,"preferredDataLocation":null,"preferredLanguage":null,"proxyAddresses":["SMTP:deniedusersforpdtdvtgen2@contosotest4.onmicrosoft.com"],"renewedDateTime":"2021-10-20T06:25:25Z","resourceProvisioningOptions":[],"securityEnabled":true,"securityIdentifier":"S-1-12-1-776682314-1181511479-1788287906-1519276500","theme":null,"visibility":null,"ODataType":"#microsoft.graph.group","pObjectId":"2e4b3b4a-6f37-466c-a21b-976ad4518e5a","ptenant":"537d6c63-efb6-4922-8643-17921eb1b0dd"}{"id":"f026a7a2-c1ed-4d10-bfd2-df5772a0d3f4","deletedDateTime":null,"classification":null,"createdDateTime":"2018-07-16T12:18:55Z","description":"Groupa20c15f8-d466-4ad3-a650-
+4df826af4e22","displayName":"Groupa20c15f8","expirationDateTime":null,"groupTypes":["Unified"],"isAssignableToRole":null,"mail":"groupa20c15f8@contosotest4.onmicrosoft.com","mailEnabled":true,"mailNickname":"groupa20c15f8","membershipRule":null,"membershipRuleProcessingState":null,"onPremisesDomainName":null,"onPremisesLastSyncDateTime":null,"onPremisesSyncEnabled":null,"preferredDataLocation":null,"preferredLanguage":null,"proxyAddresses":["SPO:SPO_70e575f2-820c-4da1-832b-2118c56a4be8@SPO_537d6c63-efb6-4922-8643-17921eb1b0dd","SMTP:groupa20c15f8@contosotest4.onmicrosoft.com"],"renewedDateTime":"2018-07-16T12:18:55Z","resourceProvisioningOptions":[],"securityEnabled":false,"securityIdentifier":"S-1-12-1-4029065122-1292943853-1474286271-4107509874","theme":null,"visibility":"Public","ODataType":"#microsoft.graph.group","pObjectId":"f026a7a2-c1ed-4d10-bfd2-df5772a0d3f4","ptenant":"537d6c63-efb6-4922-8643-17921eb1b0dd"}
```

Comment:
I am seeing some of the sample jsons like this one having red marked lines, these could be improper jsons formed due to maybe a newline.
Please validate all sample data where-ever these red lined jsons are present as they may be malformed jsons

Reply:
Adressed all the relevant files in the latest commit